app/v3/controllers/AmendLossClaimTypeController.scala

```diff
@@ -95,9 +97,9 @@ class AmendLossClaimController @Inject()(val authService: EnrolmentsAuthService,
     }
   }
 
-  private def auditSubmission(details: AmendLossClaimAuditDetail)
+  private def auditSubmission(details: AmendLossClaimTypeAuditDetail)
                              (implicit hc: HeaderCarrier,
-                              ec: ExecutionContext) = {
+                              ec: ExecutionContext): Future[AuditResult] = {
     val event = AuditEvent("amendLossClaim", "amend-loss-claim", details)
```

Comment:
would the audit name need changing (amendLossClaim -> amendLossCLaimType), or is that a change for another time, if at all?

Reply:
yep, I missed this. will change